components/contribution-flow/StepPayment.js

```diff
@@ -308,9 +329,12 @@ class StepPayment extends React.Component {
                   {icon}
                 </Flex>
                 <Flex flexDirection="column">
-                  <P fontWeight={subtitle ? 600 : 400} color="black.900">
-                    {title}
-                  </P>
+                  <div>
+                    <P display="inline" fontWeight={subtitle ? 600 : 400} color="black.900" marginRight="10px">
+                      {title}
+                    </P>
+                    {isExpired && <StyledTag display="inline">Expired</StyledTag>}
```

Comment:
As per https://docs.opencollective.com/help/contributing/development/best-practice-guidelines, strings must be internationalized. See https://docs.opencollective.com/help/contributing/development/translations.

Reply:
Okay